### ISSUE TITLE
Support for Mailgun

### ISSUE THREAD

**Issue by Nowaker** [codeorangetx/slack-to-email#6]
Mailgun offers 10,000 free emails a month. Sendgrid doesn't have a free plan at all.

**Comment by mzrzvi** [comment]
@Nowaker  I think it was free when I wrote this. I'll switch to Mailgun soon

**Comment by Nowaker** [comment]
I'd propose to add support for Mailgun. Like, use Sendgrid when `SENDGRID_API_KEY`, and use Mailgun when `MAILGUN_API_KEY` is passed.

**Comment by mzrzvi** [comment]
Makes sense -- maybe have a config env var for either

**Comment by mzrzvi** [comment]
@Nowaker Would you be interested in making a pull request?

**Comment by Nowaker** [comment]
@mzrzvi Not really. If I had to jump into the code, I'd probably redo this as `ruby-slack-to-email`. :-)

**Comment by Nowaker** [comment]
Or rather than deal with any services, would do it as an AWS Lambda thing and provision with Terraform. https://medium.com/@pixelcodeuk/create-a-slack-slash-command-with-aws-lambda-83fb172f9a74

**Comment by mzrzvi** [comment]
That'd be smart. I've been writing a bit of Lambda myself. I can make it configurable in just a few seconds really! I'll let you know when it's done

Moiz

tel:(832)%20397-9003

On Mon, Jul 24, 2017 at 6:01 PM Damian Nowak

<
mailto:Damian Nowak <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

Or rather than deal with any services, would do it as an AWS Lambda thing.
https://medium.com/@pixelcodeuk/create-a-slack-slash-command-with-aws-lambda-83fb172f9a74

—

You are receiving this because you were mentioned.

Reply to this email directly,
https://github.com/codeorangetx/slack-to-email/issues/6#issuecomment-317567678
, or
